routes/authentication.js

```diff
@@ -10,7 +10,7 @@ module.exports = function () {
     //GET login
     router.get('/login', function(req, res, next) {
 
-        res.redirect('http://localhost:8080/authenticate/atlas');
+        res.redirect('http://localhost:3001/authenticate/atlas');
```

Comment:
I don't think so the port number refers to where the authentication us done(here to the locally running mock openmrs ID)